packs/agents-of-edgewatch-bestiary/amateur-chemist.json

```diff
@@ -550,7 +1023,17 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "damageCategory": "splash",
+                        "key": "FlatModifier",
+                        "predicate": [
+                            "item:trait:splash"
+                        ],
+                        "selector": "alchemical-bomb-damage",
+                        "value": 2
```

Comment:
Went with a RE instead of just modifying the strike's splash partial so that it's properly attributed. I don't believe there's a way to override the base splash partial in a RE, so just add 2 to the base splash to make 4.